src/assets/images/EditorChoice.svg

```diff
@@ -0,0 +1,17 @@
+<path d="M20.8362 17.5513C21.7539 16.8468 22.7418 15.8467 22.7418 13.555C22.7418 13.3463 22.7418 13.1202 22.7418 12.9202C21.3851 13.8594 20.2566 14.9726 20.8362 17.5513Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M27.9812 17.2906C26.9274 18.0559 25.0613 18.2646 23.2654 18.2646C22.6288 18.2646 22.0009 18.2646 21.4257 18.2124C22.2248 17.1514 23.4806 16.7644 24.8417 16.7644C25.9095 16.7768 26.9688 16.9543 27.9812 17.2906Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M27.0238 9.39111C27.4014 10.5087 27.5902 13.6743 23.454 15.2528C23.454 13.3004 24.1829 10.4347 27.0238 9.39111Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M32.6589 14.6073C30.964 16.2815 28.8959 16.4685 25.2998 15.1683C28.1451 13.1158 31.5304 13.8638 32.6589 14.6073Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M31.9056 7.15967C32.2832 9.39477 30.0439 11.2298 28.3007 11.943C28.5378 9.39477 30.1668 7.67279 31.9056 7.15967Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M35.4776 10.241C34.5994 11.8891 31.9166 12.2891 29.7168 12.5674C30.4069 11.8193 31.2494 11.2247 32.1887 10.8229C33.1281 10.4211 34.1428 10.2213 35.1659 10.2367L35.4776 10.241Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M32.9541 9.71444C33.3098 5.94868 39.1013 5.67038 39.3472 0C40.5459 7.39236 34.8949 8.86649 32.9541 9.71444Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M18.683 17.5513C17.7653 16.8468 16.7773 15.8467 16.7773 13.555C16.7773 13.3463 16.7773 13.1202 16.7773 12.9202C18.1341 13.8594 19.2626 14.9726 18.683 17.5513Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M11.5302 17.2906C12.584 18.0559 14.4501 18.2646 16.246 18.2646C16.8827 18.2646 17.5106 18.2646 18.0858 18.2124C17.2866 17.1514 16.0308 16.7644 14.6697 16.7644C13.6019 16.7768 12.5426 16.9543 11.5302 17.2906Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M12.495 9.39111C12.1174 10.5087 11.9286 13.6743 16.0648 15.2528C16.0648 13.3004 15.3359 10.4347 12.495 9.39111Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M6.85242 14.6073C8.54729 16.2815 10.6154 16.4685 14.2115 15.1683C11.3662 13.1158 7.98087 13.8638 6.85242 14.6073Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M7.60077 7.15967C7.22316 9.39477 9.4625 11.2298 11.2057 11.943C10.9686 9.39477 9.33955 7.67279 7.60077 7.15967Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M4.03809 10.241C4.91626 11.8891 7.59907 12.2891 9.7989 12.5674C9.10881 11.8193 8.26627 11.2247 7.32695 10.8229C6.38763 10.4211 5.37291 10.2213 4.34984 10.2367L4.03809 10.241Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M6.55833 9.71444C6.20267 5.94868 0.411124 5.65298 0.165235 0C-1.03347 7.39236 4.61757 8.86649 6.55833 9.71444Z" fill="#9469FE" fill-opacity="0.75"/>
+<path d="M19.8721 20.0001C20.3911 20.0001 20.8118 19.5834 20.8118 19.0695C20.8118 18.5555 20.3911 18.1389 19.8721 18.1389C19.3532 18.1389 18.9325 18.5555 18.9325 19.0695C18.9325 19.5834 19.3532 20.0001 19.8721 20.0001Z" fill="#9469FE" fill-opacity="0.75"/>
+</svg>
```

Comment:
can we change all the `fill`'s to `fill="current"` so we can set it dynamically on the component?

Reply:
same as my comment above

Reply:
Changed